crates/bevy_pbr/src/light.rs

```diff
@@ -23,6 +25,8 @@ impl Default for Light {
             color: Color::rgb(1.0, 1.0, 1.0),
             depth: 0.1..50.0,
             fov: f32::to_radians(60.0),
+            intensity: 100.0,
+            range: 10.0,
```

Comment:
I saw you changed the range of some examples to 20.0, would that maybe be a better default?

Reply:
Maybe? It's a bit arbitrary, but sure. What do people think?

Reply:
it makes sense to use it as the new default, so that the examples stay simple without having to change it

Reply:
Will do.

Reply:
Done.